com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -309,7 +309,7 @@ static bool TryGetCullingParameters(CameraData cameraData, out ScriptableCulling
                 cullingParams = cameraData.xr.cullingParams;
 
                 // Sync the FOV on the camera to match the projection from the XR device
-                if (!cameraData.camera.usePhysicalProperties)
+                if (!cameraData.camera.usePhysicalProperties && !XRGraphicsAutomatedTests.enabled)
```

Comment:
is there a reason why we don't want to sync up fov when automation is enabled?

Reply:
HDRP has a similar change. It's because the test framework will override the culling params and will get in a kind of feedback loop. This might hide a deeper issue. We should investigate further once we have everything working.